src/workers/personalizedDigestEmail.ts

```diff
@@ -141,6 +144,24 @@ const digestTypeToFunctionMap: Record<
       },
     );
   },
+  [UserPersonalizedDigestType.ReadingStreakReminder]: async (data, con) => {
```

Comment:
Lol we should probably rename this worker 😂

Reply:
Yeah, probably 😬
Having the streak and reading reminders in personalized digest doesn't make sense. We could probably refactor it into to tables too

Reply:
I guess the one table works, but the naming is a bit confusing maybe 🤔
(not urgent/blocking)

But more generic would probably be nice.